### DISCUSSION TITLE
Model loads completely black on Chrome and Firefox

### DISCUSSION THREAD

**Discussion by thebetterdraper** [google/model-viewer#2746] [Q&A]
The .glb file I have loads completely fine in GLB Viewers online, but loading into model-viewer causes it to appear with (What I assume is) absolutely no ambient lighting. There have been similiar bugs mentioned in the discussions, but specific to Safari and endemic to something in their recent update.

This one is straight up refusing to load any lighting. But it's fine elsewhere:

I have attached screenshots here, and I will attach a link to the model here too.

Images of what it looks like in the model editor:
![image](https://user-images.githubusercontent.com/43851563/131575415-bc994038-2982-4820-8817-7b1942449866.png)
![image](https://user-images.githubusercontent.com/43851563/131575494-a4933876-c2f7-475d-a71b-9ecb323cde14.png)

Can someone help out here? I can't figure out why this would happen, especially given that every other model in glb as well as gltf format is loading just fine. 

Pasting my snippet here, but it's pretty unchanged:

<model-viewer 
                ref={model}
                id="helmetModelViewer" 
                camera-controls 
                alt="A 3D model of an Helmet" 
                // src={modelSrc}
                src={LocalModel}
                // auto-rotate
                loading="eager"
                camera-orbit = "110deg 85deg 105%"
                max-camera-orbit = "200deg 85deg 155%"
                min-camera-orbit = "40deg 85deg 155%"
                disable-zoom
                
                >
                </model-viewer>

**Comment by elalish** [comment]
Your link doesn't seem to be working. If you can update it, I'll take a look. Does the validator show any warnings?

**Reply by thebetterdraper** [reply]
> Your link doesn't seem to be working. If you can update it, I'll take a look. Does the validator show any warnings?



Sorry, I'll fix the link. And no the validator shows nothing. For some reason, this is the only model I have faced this issue with, yet the same model loads well everywhere else.

I adjusted directional light intensity in a few other viewers and was able to repro this. It was only happening when directional intensity was 0.

I don't know how to adjust that in model-viewer though.

**Reply by elalish** [reply]
Ah, I bet I know the problem! We don't use a directional light, only environmental. A common glTF exporter bug is to set the AO channel to all black (zero), which stops all environmental light, but not directional. I bet fixing your AO channel will fix this. It should be one by default.

**Reply by thebetterdraper** [reply]
> Ah, I bet I know the problem! We don't use a directional light, only environmental. A common glTF exporter bug is to set the AO channel to all black (zero), which stops all environmental light, but not directional. I bet fixing your AO channel will fix this. It should be one by default.

Hey thanks so much for pointing this out. I am a complete novice at all things 3d on the web. Could you clue me in on how to change the AO value? I looked at the accompanying JSON file and found this:

"shaders": {
        "Main shader": {
            "parameters": {
                "Common Parameters": {
                    "ao_intensity": 0.75,
                    "emissive_intensity": 1,
                    "horizonFade": 1.2999999523162842,
                    "nbSamples": 16
                },


Is "ao_intensity" what you mean ? Or should I be looking somewhere else?

**Reply by elalish** [reply]
I have no idea what you're looking at, but it's not glTF JSON. Generally the problem is in your AO  texture, which is usually combined with metallic roughness. You'll have to look into the tool you're  using for glTF export.

**Reply by thebetterdraper** [reply]
Hey @elalish  just wanted to tell you I figured it out, thanks to what you said about AO. The fix was to go into modeviewer's editor, load the glb, and play around with the occlusion textures until the individual profiles began to be visible. Thank you so much! I hope anyone facing the same issue can use my fix.
